app-frontend/src/app/components/scenes/sceneItem/sceneItem.module.js

```diff
@@ -80,6 +80,14 @@ class SceneItemController {
             }
         );
     }
+
+    hasDlPerm() {
+        if (this.scene.planetPerm) {
+            this.permInfo = 'This imagery requires additional access from planet.com.';
```

Comment:
I'm trying to keep anything repository specific out of the scene item / browse view.
We should add a function to each of the repositories that returns permissions for the scenes using the same interface.
I can go over it with you today.